system/jlib/jexcept.hpp

```diff
@@ -129,6 +129,7 @@ void  jlib_decl disableSEHtoExceptionMapping();
 
 void jlib_decl *setSEHtoExceptionHandler(IExceptionHandler *handler); // sets handler and return old value
 
+void jlib_decl raiseKillSigInFuture(unsigned timeoutSec);
```

Comment:
minor: No need to be in the header, or jlib_decl.

Reply:
ok, changed